scripts/init.sh

```diff
@@ -24,10 +24,34 @@ for var in "${vars_to_check[@]}"; do
   export "${var}"
 done
 
+# Setting the Node Role
+env_file_exist "${ROOT_DIR}/${ENV_FILE}"
+# shellcheck disable=SC1090
+source "${ROOT_DIR}/${ENV_FILE}" || { echo "Error: could not source ${ROOT_DIR}/${ENV_FILE} file. Fix it before proceeding any further.  Exiting..."; exit 1; }
+
+if [ -z "${SCNODE_ROLE:-}" ]; then
+  read -rp "Which kind of node would you like to start ? Choose between 'rpc' or 'forger': " SCNODE_ROLE_VALUE
```

Comment:
"What kind of node type would you like to run('rpc' or 'forger'): 

SCNODE_ROLE_VALUE should be smaller case since it is LOCAL var. Check all the occurences